daffodil-core/src/main/scala/org/apache/daffodil/runtime1/VariableRuntime1Mixin.scala

```diff
@@ -102,6 +102,8 @@ trait DFDLNewVariableInstanceRuntime1Mixin { self: DFDLNewVariableInstance =>
 
     Maybe.toMaybe(defaultValExpr)
   }
+
+  val maybeDefaultValueExprDelay = Delay(maybeDefaultValueExpr, 'maybeDefaultValueExpr2)
```

Comment:
Is this needed anymore? Or can you go back to using `Delay(maybeDefaultValueExpr)` and have the VRD.initialize method force the delay to be evaluated?

Reply:
yup. I'm fixing this so it matches the style of setVariable.